temp_monitor.py

```
@@ -161,24 +227,64 @@ def get_humidity():
             )
-            
+
+            # Check thresholds and send alerts via webhook
+            if webhook_service:
+                try:
+                    alerts_sent = webhook_service.check_and_alert(
+                        current_temp, current_humidity, last_updated
+                    )
+                    if alerts_sent:
+                        logging.info(f"Webhook alerts sent: {list(alerts_sent.keys())}")
+                except Exception as webhook_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)